scripts/migrate_operations_to_workflows.py

```
@@ -0,0 +1,277 @@
+        
+        # Scan source table
+        logger.info("Scanning source table...")
+        try:
+            source_items = self.scan_source_table(batch_size)
+            logger.info(f"Found {len(source_items)} items to migrate")
+            
+            if len(source_items) == 0:
+                logger.info("No items to migrate")
+                return True
+                
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)